### DISCUSSION TITLE
MFXProgressBar set background-radius does not work

### DISCUSSION THREAD

**Discussion by Lairon1** [palexdev/MaterialFX#335] [Q&A] [answered]
I'm trying to round the edges of an MFXProgressBar, but the -fx-background-radius and -fx-border-radius tags don't work.
I tried to specify them already where possible, but I do not get any changes.

This is what my css file looks like now.

```
.mfx-progress-bar .bar1 {
    -fx-fill: linear-gradient(from 0% 50% to 100% 50%, #18B14F 0%, #18B14F 75%, #50C82F 100%);
}

.mfx-progress-bar .bar2 {
    -fx-fill: linear-gradient(from 0% 50% to 100% 50%, #18B14F 0%, #18B14F 75%, #50C82F 100%) ;
}

.mfx-progress-bar .track {
    -fx-fill: #303030;
}
```

On the page with the demo project, the ProgressBar has rounded edges, but no matter how much I climbed the code of the demo project, I did not find how to do it

**UPD:** I noticed that in Scene Builder rounded edges are displayed even if I don't set them. But it doesn't work on the real window.

**Comment by palexdev** [comment, marked as answer]
1) I believe you didn't apply the MaterialFX theme
2) You didn't find it in the demo code because the radius is set by default. It's here: [MFXProgressBar](https://github.com/palexdev/MaterialFX/blob/main/materialfx/src/main/resources/io/github/palexdev/materialfx/css/MFXProgressBar.css)
